src/main/java/uk/gov/ida/matchingserviceadapter/controllogic/UnknownUserAttributeQueryHandler.java

```diff
@@ -59,14 +59,6 @@ public UnknownUserAttributeQueryHandler(
     }
 
     public OutboundResponseFromUnknownUserCreationService createNewVerifyAccount(InboundVerifyMatchingServiceRequest attributeQuery) {
```

Comment:
You might like to rename this to make it clear it handles both eIDAS and Verify flows.